code/modules/cooking/machinery/cooking_machines/container.dm

```diff
@@ -249,10 +249,10 @@
 	return TRUE
 
 /obj/item/reagent_containers/cooking_container/plate
```

Comment:
You should change the path to reflect the new name.
Otherwise it can be confusing if you want to search for certain things.